.changeset/hot-eyes-knock.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/secondary-button': patch
+'@commercetools-uikit/design-system': patch
+---
+
+Add tone: "info" for the secondary button
```

Comment:
```suggestion

A new property called **tone** has been added to the component so consumers can use it with different color schemes (**secondary** and **info** by now).

Alto, the **theme** property has been deprecated and it will be removed in the future, so please don't use it anymore (use the new one instead if needed).
```

Reply:
Updated [149f6b0](https://github.com/commercetools/ui-kit/pull/2576/commits/149f6b03b46e9b186402680cf45333b747032353)